bin/sidekiq_wrapper

```diff
@@ -5,17 +5,20 @@ require_relative '../lib/environment_light.rb'
 
 RAILS_ROOT = File.expand_path('../..', __FILE__).freeze
 SIDEKIQ_EXECUTABLE = File.join(RAILS_ROOT, 'bin/sidekiq').freeze
-SIDEKIQ_BASE_ARGUMENTS =
-  %W(#{SIDEKIQ_EXECUTABLE}
-     --environment #{ENV['RAILS_ENV'] || 'production'}).freeze
 
 # Parse options
 def parse_options
-  options = {workdir: Dir.pwd,
+  options = {environment: 'production',
+             workdir: Dir.pwd,
              timeout: 15}
   OptionParser.new do |opts|
     opts.banner = "Usage: #{File.basename(__FILE__)} [options]"
 
+    opts.on('-eMANDATORY', '--environment=MANDATORY',
+            "Set Rails environment (default: #{options[:environment]})") do |e|
+      options[:environment] = e
+    end
+
     opts.on('-wMANDATORY', '--workdir=MANDATORY',
             'Set work directory (default: CWD)') do |w|
```

Comment:
Admins usually have no notion of ill/senseless optparse documentation. However, they know [indirectly] http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html by reading man pages and usage synopsis from native Unix utilities ...